src/components/cc-global-header.vue

```diff
@@ -128,11 +128,42 @@ export default defineComponent({
       ];
     } else {
       vm.menuLoading = true;
+      axios.interceptors.response.use(
+        undefined,
+        function retryApiRequest(error) {
+          var config = error.config;
+          // If config does not exist or the retry option is not set, reject
+          if (!config || !config.retry) return Promise.reject(error);
+
```

Comment:
Somewhere around here, let's define an explanatory variable for the `defaultRetryDelay`

```suggestion
// Set the default retry delay in milliseconds
const defaultRetryDelay = 1000;
```